src/__tests__/Carousel.js

```diff
@@ -99,13 +103,17 @@ describe("Slider", function() {
         describe("Initial State", () => {
             const props = {
                 selectedItem: 0,
-                hasMount: false
+                hasMount: false,
+                loop: false,
+                swipeLoop: false
```

Comment:
I'm struggling with the definition of these two: "loop" and "swipeLoop" - some comments would help greatly.